prompb/rpc.proto

```diff
@@ -61,3 +62,56 @@ message SeriesDeleteRequest {
 
 message SeriesDeleteResponse {
 }
+
+service Querying {
+  // ListSeries lists all series matching one of the given label selectors.
+  rpc ListSeries(ListSeriesRequest) returns (ListSeriesResponse);
+
+  // ListLabelValues lists all known for a single label.
+  rpc ListLabelValues(LabelValuesRequest) returns (LabelValuesResponse);
+
+  // QueryAtTime runs a query at a single point in time.
+  rpc QueryAtTime(QueryAtTimeRequest) returns (QueryResponse);
+
+  // QueryOverRange runs a query at a series of times of in a time range.
+  rpc QueryOverRange(QueryOverRangeRequest) returns (QueryResponse);
+}
+
+message ListSeriesRequest {
+  google.protobuf.Timestamp start_time = 1 [(gogoproto.stdtime) = true];
+  google.protobuf.Timestamp end_time   = 2 [(gogoproto.stdtime) = true];
+  repeated SeriesMatcher matchers      = 3 [(gogoproto.nullable) = false];
+}
+
+message ListSeriesResponse {
+  repeated Labels series = 1;
+}
+
+message LabelValuesRequest {
+  string name = 1;
+}
+message LabelValuesResponse {
+  repeated string values = 1;
+}
+
+message QueryAtTimeRequest {
+  google.protobuf.Timestamp time   = 1 [(gogoproto.stdtime) = true];
+  google.protobuf.Duration timeout = 2 [(gogoproto.stdduration) = true];
```

Comment:
Should this still be here with grpc? Seconds are also what the existing api uses.

Reply:
I was wondering about that.  In the remote apis we take ms as an int64; in the delete series admin endpoint we use a Timestamp.

I think we should use a timestamp here, as its more 'semantic', and just ensure the json encoding accepts seconds as a float (or whatever the existing api is).